src/app/users/utils.js

```diff
@@ -213,8 +213,20 @@ const mapUserToSupportModel = (user, userFromSearch) => {
   };
 };
 
+const checkManageAccess = async (arr) => {
+    const manage = await getServiceById('manage') 
+    let userServiceIds = []
+
+    for (let i = 0; i < arr.length; i++) {
+      userServiceIds.push(arr[i].serviceId)
+    }
+
+    return userServiceIds.includes(manage.id)
```

Comment:
Similar sort of thing:
```
const userServiceIds = arr.map(entry => entry.serviceId);
```
Although it seems like you are building an array just to search it; so perhaps even:
```
return arr.some(entry => entry.serviceId === manage.id);
```

Reply:
Thanks Lea, need to brush up on my array methods 👍